Azure/sonic-buildimage
# [sonic-cfggen]: Update UT to add port lanes

Signed-off-by: Gang Lv ganglv@microsoft.com



#### Why I did it
Need to run yang validation for sonic-cfggen unit test, and many unit test does not provide lanes for port table.

#### How I did it
Update port config file.

#### How to verify it
Run sonic-cfggen unit test,
Use below PR to verify 
https://github.com/Azure/sonic-buildimage/pull/10228/

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

Fix #9553


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
